docs/content/concepts/assets/asset-observations.mdx

```diff
@@ -75,7 +75,6 @@ def observes_dataset_op(context: OpExecutionContext):
             },
         )
     )
-    context.log_event(AssetMaterialization(asset_key="my_dataset"))
     return remote_storage_path
 ```
 
```

Comment:
didn't update the screenshot cos i think it's ok to leave the materialization there to show they both can be associated to the same asset. plus the screenshot does focus on the observation